google-factcheck-explorer-bot/README.md

```diff
@@ -0,0 +1,79 @@
+# Google factcheck explorer bot
```

Comment:
This is a public repository, so I think we should hide all internal links from it.

Reply:
Good point. Seems like these links won't be useful externally, but are they harmful?  Should I copy the content on "how to configure a webhook" into the readme so that people outside the org will be able to do it?